docs/testing-guidelines/WritingPesterTests.md

```diff
@@ -51,24 +51,18 @@ Describe "One is really one" {
 }
 ```
 
-If you are checking for proper errors, do that in a `try/catch`, and then check `FullyQualifiedErrorId`. Checking against `FullyQualifiedErrorId` is recommended because it does not change based on culture as an error message might.
+If you are checking for proper errors, use the `ShouldBeErrorId` helper defined in HelpersCommon.psm1 module which is in your path if you import `build.psm1`.
+Checking against `FullyQualifiedErrorId` is recommended because it does not change based on culture as an error message might.
 
 ```powershell
 ...
-it "Get-Item on a nonexisting file should have error PathNotFound" {
```

Comment:
[Here](https://github.com/PowerShell/PowerShell/blob/ffd39b2853b68419eb0fd5e34a89b755c98b8022/test/powershell/engine/Remoting/SSHRemotingAPI.Tests.ps1#L63) and [here](https://github.com/PowerShell/PowerShell/blob/02b5f357a20e6dee9f8e60e3adb9025be3c94490/test/powershell/Language/Scripting/HashtableToPSCustomObjectConversion.Tests.ps1#L106) we didn't use the pattern.